compiler/native/transform.go

```diff
@@ -18,12 +18,15 @@ const (
 	// default ID for steps in a stage in a pipeline.
 	// format: `<org name>_<repo name>_<build number>_<stage name>_<step name>`
 	stageID = "%s_%s_%d_%s_%s"
-	// default ID for steps in a pipeline.
+	// default ID for stage steps in a pipeline.
```

Comment:
Are we sure we want to add this here? 👍 

I ask because I think the `stepID` variable isn't used for stage steps

Reply:
I can remove it, I thought there was a pattern in place when I was reading the comments.